berkeley-function-call-leaderboard/bfcl/eval_checker/model_metadata.py

```diff
@@ -518,28 +518,16 @@
         "Meta Llama 3 Community",
     ],
     "command-r-plus-FC": [
-        "Command-R-Plus (FC) (Original)",
```

Comment:
Since there's no longer an optimised version of command-r-plus I thought it would be beneficial to drop the `(Original)` from the name. I'd be happy to add it back if you think otherwise, though!